java/test/jmri/jmrix/pi/RaspberryPiSensorManagerTest.java

```diff
@@ -109,7 +109,14 @@ public void testProvideName() {
         Assert.assertTrue("real object returned ", t != null);
         Assert.assertTrue("system name correct ", t == l.getBySystemName(getSystemName(14)));
     }
-    
+
+    @Test
+    @Ignore
+    @Override
+    public void testRegisterDuplicateSystemName() {
+        // This test doesn't work for this class.
```

Comment:
Please put the reason as an argument to the ignore.

Reply:
Do you mean
`@Ignore("This test doesn't work for this class")`
or an explanation on why the test doesn't work? The reason is that the library register I/O pins for the turnout/sensor which when resides when other tests are run.

Reply:
Yes, that is what I mean.  

And in addition, you should add a jmri.util.junit.annotations.ToDo annotation with a comment that says the setup needs to be fixed to enable the test.

Unfortunately, the library itself maintains static references to allocated pins, and it doesn't always cleanly clear the references correctly.  (but that is a library problem, not our problem).

Reply:
I have updated the @Ignore and added a @ToDo for both tests.